src/containers/Tenant/utils/schema.ts

```diff
@@ -1,10 +1,23 @@
-import type {NavigationTreeNodeType} from "ydb-ui-components";
+import type {NavigationTreeNodeType} from 'ydb-ui-components';
+import {EPathType} from '../../../types/api/schema';
 
-const DB_TYPES = new Set(['EPathTypeSubDomain']);
-const TABLE_TYPES = new Set(['EPathTypeTable', 'EPathTypeOlapTable']);
-const DIR_TYPES = new Set(['EPathTypeDir', 'EPathTypeOlapStore']);
+const DB_TYPES = new Set([
+    EPathType.EPathTypeSubDomain,
+]);
+const TABLE_TYPES = new Set([
+    EPathType.EPathTypeTable,
+    EPathType.EPathTypeColumnTable,
+]);
+const DIR_TYPES = new Set([
+    EPathType.EPathTypeDir,
+    EPathType.EPathTypeColumnStore,
+]);
+
+export const calcNavigationTreeType = (type?: EPathType): NavigationTreeNodeType => {
+    if (!type) {
+        return 'directory';
+    }
 
-export const calcNavigationTreeType = (type: string): NavigationTreeNodeType => {
     if (DIR_TYPES.has(type)) {
```

Comment:
There are not much types, so I think it would be more clear to use `switch ... case` instead of sets.

Reply:
Why?
The sets names are clear, and adding new types only requires extending a set

Reply:
I'll modify this function a bit though, better have a default value than checking `type` for existence

Reply:
I see you point though
`switch...case` also requires only one modification to extend the types
my concern is the amount of types will grow and the `switch...case` will become unreadable
I'm not sure whether this would happen though